crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -20,3 +25,103 @@ async fn create_job_fails_job_already_exists() {
 async fn create_job_fails_works_new_job() {
     // TODO
 }
+
+use std::str::FromStr;
+
+impl FromStr for JobStatus {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "Created" => Ok(JobStatus::Created),
+            "LockedForProcessing" => Ok(JobStatus::LockedForProcessing),
+            "PendingVerification" => Ok(JobStatus::PendingVerification),
+            "Completed" => Ok(JobStatus::Completed),
+            "VerificationTimeout" => Ok(JobStatus::VerificationTimeout),
+            "Failed" => Ok(JobStatus::Failed),
+            s if s.starts_with("VerificationFailed(") && s.ends_with(')') => {
```

Comment:
didn't we remove the reason in `VerificationFailed`?

Reply:
Yes, 
you are correct.
It was removed as a part of #52, rebased/removed.